Github/sarif-output.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Sarif
```

Comment:
Input more descriptive name - i.e. "Checkmarx Sarif Integration"

Reply:
Changed the name for the action